Documentation/Blazorise.Docs/Pages/Docs/Extensions/PdfViewer/PdfViewerPage.razor

```diff
@@ -0,0 +1,176 @@
+﻿@page "/docs/extensions/pdfviewer"
+
+<Seo Canonical="/docs/extensions/pdfviewer" Title="Blazorise PdfViewer Component" Description="Learn to use and work with the Blazorise PdfViewer component, a component used to display a regular PDF documents." />
+
+<DocsPageTitle Path="Extensions/PdfViewer">
+    Blazorise PdfViewer Component
+</DocsPageTitle>
+
+<DocsPageLead>
+    A <Code>PdfViewer</Code> component used to display a regular PDF document.
+</DocsPageLead>
+
+<DocsPageSubtitle>
+    Overview
+</DocsPageSubtitle>
+
+<DocsPageParagraph>
+    This documentation covers the integration of a PDF viewer with a toolbar in a Blazor application using the <Strong>PdfViewerContainer</Strong>, <Strong>PdfViewerToolbar</Strong>, and <Strong>PdfViewer</Strong> components. The <Strong>PdfViewerState</Strong> class is utilized to manage the state and events associated with the PDF viewer. This setup provides a complete solution for displaying and interacting with PDF documents.
```

Comment:
It's just bellow this sentence.

Reply:
You're right, in this case then this should be a brief introduction of what it does regardless of whatever components it uses behind the covers.
The next section goes into the components being used. In my opinion at least. Otherwise it's fine.